docs/admin/backup.rst

```diff
@@ -31,29 +29,29 @@ Dumped data for backups
 
 Stored in :setting:`DATA_DIR` ``/backups``.
 
-Weblate dumps various data here and you can use the files here for better
-backups. The files are updated daily (requires running Celery beats server, see
+Weblate dumps various data here, and you can include these files for more complete
+backups. The files are updated daily (requires a running Celery beats server, see
 :ref:`celery`).  Currently this includes:
 
-* Translation memory dump in JSON format.
+* Translation memory dump, in JSON format.
 
 Version control repositories
 ++++++++++++++++++++++++++++
 
 Stored in :setting:`DATA_DIR` ``/vcs``.
 
-The version control repositories contain copy of your upstream repositories
-with Weblate changes. If you have push on commit enabled on all your
-translation components, then all Weblate changes are included upstream and you
-do not have to backup the repositories on Weblate side. They can be cloned
-again from upstream locations with no data loss.
+The version control repositories contain a copy of your upstream repositories
+with Weblate changes. If you have push on commit enabled for all your
+translation components, all Weblate changes are included upstream and you
+do not have to backup the repositories on the Weblate side. They can be cloned
+again from the upstream locations with no data loss.
 
 SSH and GPG keys
 ++++++++++++++++
 
 Stored in :setting:`DATA_DIR` ``/ssh`` and :setting:`DATA_DIR` ``/home``.
 
-If you are using Weblate generated SSH or GPG keys, you should backup these
+If you are using SSH or GPG keys generated by Weblate, you should back up these
```

Comment:
This is not clear. The locations they are stored in? The part about the private keys I don't understand at all.